src/elements/notification/notification.scss

```diff
@@ -196,6 +196,8 @@ $open-anim-height-to: calc(
 .sbb-notification__icon {
   color: var(--sbb-notification-icon-color);
   margin-block-start: var(--sbb-notification-icon-margin-block-start);
+
+  --sbb-icon-svg-width: var(--sbb-notification-icon-size);
```

Comment:
As we don't document this, we should add a comment here that this is here on purpose and why. Otherwise, it feels like a code smell that can be removed (nowhere used, not publicly documented).